src/main/java/com/tickets/model/MovieSession.java

```diff
@@ -0,0 +1,62 @@
+package com.tickets.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
```

Comment:
let's use let's use `@JoinColumn` annotation so that our columns would have the `right_naming` - and not `camelCaseName`